playground/css/vite.config.js

```diff
@@ -69,15 +69,15 @@ export default defineConfig({
           function (url) {
             return url === 'virtual-file-absolute'
               ? {
-                  contents: `@import "${pathToFileURL(path.join(import.meta.dirname, 'file-absolute.scss')).href}"`,
+                  contents: `@use "${pathToFileURL(path.join(import.meta.dirname, 'file-absolute.scss')).href}"`,
                 }
               : null
           },
           function (url) {
             return url.endsWith('.wxss') ? { contents: '' } : null
           },
         ],
-        silenceDeprecations: ['legacy-js-api'],
+        silenceDeprecations: ['legacy-js-api', 'import'],
```

Comment:
I added `import` to silenceDeprecations due to https://github.com/sass/embedded-host-node/issues/340.